File: app/src/main/java/com/quickstart/api/GitHubService.java
package com.quickstart.api;
import com.quickstart.models.Repo;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;


/**
 * Created at 06.04.16 17:06
 *
 * @author Alexey_Ivanov
 */
public interface GitHubService {
    @GET("users/{user}/repos")
    Call<List<Repo>> listRepos(@Path("user") String user);
}

File: app/src/main/kotlin/com/quickstart/api/GitHubService.kt
package com.quickstart.api

import com.quickstart.models.Repo
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * Created at 29.11.16 11:58
 * @author Alexey_Ivanov
 */
interface GitHubService {
    @GET("users/{user}/repos")
    fun listRepos(@Path("user") user: String): Observable<List<Repo>>
}
